src/main/kotlin/nl/info/zac/app/zaak/ZaakRestService.kt

```diff
@@ -518,27 +518,23 @@ class ZaakRestService @Inject constructor(
     @Path("zaaktypes")
     fun listZaaktypes(): List<RestZaaktype> =
```

Comment:
Don't we need multiple endpoints for listing zaaktypes? This story is about listing zaaktypes for which the logged in user is authorised to create a zaak. Is this endpoint specifically used for this scenario? If not we need a new endpoint I would think? `listZaaktypesForZaakCreation` or something like that?